Snippets/Core/Core_9.1/Core_9.1.csproj

```diff
@@ -20,4 +20,4 @@
     <PackageReference Include="OpenTelemetry" Version="1.3.0" />
```

Comment:
The other PR has solution file changes that don't appear in this one, is that intentional?